.github/workflows/add-issue.yml

```diff
@@ -0,0 +1,16 @@
+name: Add Issue to Board
+
+on:
+  issues:
+    types: [opened]
+
```

Comment:
Does this not need project write permissions?

Reply:
No, because it uses the PAT below.